rhea/cores/uart/uartlite.py

```diff
@@ -1,36 +1,48 @@
 
-from myhdl import Signal, instances
-
+from myhdl import Signal, instances, always_seq, always_comb
 from ..misc import syncro
 from .uartbase import uartbaud, uarttx, uartrx
 from ..fifo import fifo_fast
+from rhea.system import FIFOBus
 
 
-def uartlite(glbl, fbustx, fbusrx, serial_in, serial_out, baudrate=115200):
+def uartlite(glbl, fifobus, serial_in, serial_out, baudrate=115200):
     """ The top-level for a minimal fixed baud UART
 
+    The UART has one external FIFOBus interface and two internal
+    ones to the specific RX/TX fifos. It is through the internal fbusrx,fbustx
+    FIFOBus interfaces that the UART reads into/writes from the RX/TX fifo queues
```

Comment:
The doc strings typically include user information and not implementation details unless is affects the user interface (API).  The information about the internal FIFO will be confusing from a user of the `uartlite`, these need to be moved to internal comment and/or developer documentation.